doc/coq.rst

```diff
@@ -134,13 +137,15 @@ The semantics of the fields are:
   Interproject composition has been available since :ref:`Coq lang
   0.4<coq-lang>`.
 
-  As of today, Dune cannot depend on installed Coq theories. This restriction
-  will be lifted in the future. Note that composition with the Coq standard
-  library is supported, but in this case the ``Coq`` prefix has been made
-  available in a qualified way, since :ref:`Coq lang 0.2<coq-lang>`.
+  Composition with installed theories has been available since :ref:`Coq lang
+  0.8<coq-lang>`. Instaled theories do not have to be installed using Dune. Dune
+  will scan for theories installed in the ``user-contrib`` directory of the Coq
+  installation or those given in the ``COQPATH`` environment variable.
 
-  You may still use installed libraries in your Coq project, but there is
-  currently no way for Dune to know about it.
+  Composition with the Coq standard library is supported, but in this case the
+  ``Coq`` prefix has been made available in a qualified way, since :ref:`Coq
```

Comment:
That seems a bit confusing to me, we should be more clear and say:
- if `(stdlib yes)`, the `Coq` is a added using -R + Require Import Prelude is assumed
- if `(stdlib no)`, then `Coq` can be added to the regular theory field, and it will be handled as a regular library.